openstudiocore/src/model/ScheduleDay.cpp

```diff
@@ -202,7 +202,7 @@ namespace detail {
 
   double ScheduleDay_Impl::getValue(const openstudio::Time& time) const
   {
-    if (time.totalDays() < 0.0 || time.totalDays() > 1.0){
+    if (time.totalMinutes() < 0.0 || time.totalDays() > 1.0){

```

Comment:
@kbenne I don't think this is right here or in ::times(), we only need this in addValue I think